test/EFCore.MySql.FunctionalTests/TestUtilities/Attributes/SupportedServerVersionFactAttribute.cs

```diff
@@ -17,7 +18,7 @@ private SupportedServerVersionFactAttribute(ServerVersionSupport serverVersionSu
 
             if (!serverVersionSupport.IsSupported(currentVersion) && string.IsNullOrEmpty(Skip))
             {
-                Skip = $"Test is supported only on {serverVersionSupport.SupportedServerVersions} and higher.";
+                Skip = $"Test is supported only on {serverVersionSupport.SupportedServerVersions.Join()} and higher.";
```

Comment:
Nice catch. We should actually be able to use `serverVersionSupport` directly, because its `ToString()` overload should take care of the concatenation.